new_array_processing.py

```diff
@@ -84,24 +83,26 @@ def get_volcano_backazimuth(latlist, lonlist, volcanoes):
             print('Reading in data from IRIS')
             client = Client("IRIS")
             try:
-
                 st = client.get_waveforms(NET, STA, LOC, CHAN,
-                                          STARTTIME, ENDTIME, attach_response=True)
+                                          STARTTIME - 2 * config.taper_val,
+                                          ENDTIME + 2 * config.taper_val,
+                                          attach_response=True)
             except header.FDSNNoDataException:
                 print(f"No data retrieved for {STA}")
                 continue
 
             st.merge(fill_value='latest')
-            st.trim(STARTTIME, ENDTIME, pad='true', fill_value=0)
+            st.trim(STARTTIME - 2 * config.taper_val, ENDTIME + 2 * config.taper_val, pad='true', fill_value=0)
```

Comment:
So you trim it to *less* than the requested start/end times? Not just *to* the requested start/end?

Reply:
Oh, wait. Got my signs backwards. That makes sense now.